x-pack/plugins/index_lifecycle_management/public/store/actions/lifecycle.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { toastNotifications } from 'ui/notify';
+import { saveLifecycle as saveLifecycleApi } from '../../services/api';
+
+
+export const saveLifecyclePolicy = (lifecycle, isNew) => async () => {
+  try {
+    await saveLifecycleApi(lifecycle);
+  }
+  catch (err) {
+    toastNotifications.addDanger(err.data.message);
```

Comment:
I added some more robust error handling for rollups in https://github.com/elastic/kibana/pull/25092. I think it's worth copying over the helpers I added in that PR and using the same patterns here. The helpers and patterns will attempt to display a toast if the error is emitted by `$http` but if it isn't then it will show a fatal error.

Reply:
done